src/taker/api.rs

```diff
@@ -1209,6 +1209,7 @@ impl Taker {
                         previous_funding_output,
                         maker_funding_tx_value,
                         next_contract_redeemscript,
+                        self.ongoing_swap_state.swap_params.fee_rate,
```

Comment:
For now this works. This fee rate is for the funding transaction. The contract transaction fee rate estimation is tricky, so will be handled later.

Reply:
oh ok :thinking: I know Rust, but I am new to Bitcoin. I am going to keep reading the docs and trying to contribute to the project :nerd_face: